go-controller/pkg/cni/types/types.go

```diff
@@ -0,0 +1,12 @@
+package types
```

Comment:
Not sure why we need this new file. Can't you move the new Netconf definition to  `go-controller/pkg/cni/types.go` itself?

Reply:
the problem is cycle import in go. so currently cni package is using confg package. if I would add it to the types in the cni package I will need to import that in the config package. so to work around it with minimal changes I added the cni/types package.

Reply:
makes sense. thanks for explaining.